### PR TITLE
Style delete button

### PR DESCRIPTION
**Pull request by nwgambee**
## Problems 
Needed to add styling to the new delete-list button that was added. 
## Solutions 
Added styling for that button and also fixed a small bug when navigating back to readingList after a deletion.
## Issues Resolved
Resolved #62

### CODE REVIEW

**Comment by nwgambee** [comment]
seems like my commit log is still screwy. Will attempt to reset that if I make another PR

**Comment by DanielEFrampton** [comment]
We tried to fix this using interactive rebasing, but ran against problems trying to squash commits without a previous commit, and then got lost in what we don't know about using Vim.

**Review by DanielEFrampton — COMMENTED** [review]
Look good!

File: app/assets/stylesheets/curated-list-items.scss [hunk 1]
```diff
@@ -144,4 +144,24 @@
       lighten(red, 10%)
     );
   }
+}
+
```
Comment (Tristan-Holtz) [inline app/assets/stylesheets/curated-list-items.scss:148]: Styling in line with site

**Review by Tristan-Holtz — APPROVED** [review]

**Pull request merged**